content/docs/command-reference/exp/pull.md

```diff
@@ -6,13 +6,13 @@ data from a `dvc remote`.
 ## Synopsis
 
 ```usage
-usage: dvc exp pull [-h] [-q | -v] [-f] [--no-cache]
-                    [-r <name>] [-j <number>] [--run-cache]
+usage: dvc exp pull [-h] [-q | -v] [-A] [--rev <commit>] [-n <num>] [-f]
+                    [--no-cache] [-r <name>] [-j <number>] [--run-cache]
                     git_remote experiment
 
 positional arguments:
   git_remote          Git remote name or Git URL
-  experiment          Experiment to pull
+  experiment          Experiments to pull
```

Comment:
If it's experiments shouldn't the usage block look different?

Reply:
This is what I get for `dvc exp pull --help`:

```
usage: dvc experiments pull [-h] [-q | -v] [-A] [--rev <commit>] [-n <num>] [-f]
                            [--no-cache] [-r <name>] [-j <number>] [--run-cache]
                            <git_remote> [<experiment> ...]
```

Is there any reason we wouldn't use that? Are others not seeing the same?

Reply:
Yes what you see is what I meant @dberenbaum , specifically the line

`                            <git_remote> [<experiment> ...]`

But if this is part of #3340 and other refs need to be updated as well, maybe don't include the change here at all.